hera_sim/visibilities/conversions.py

```diff
@@ -105,7 +106,6 @@ def healpix_to_crd_eq(h, nest=False):
     assert h.ndim == 1, "h must be a 1D array."
 
     px = np.arange(len(h))
-    crd_eq = np.array(
-        healpy.pix2vec(healpy.get_nside(h), px, nest=nest), dtype=np.float32
+    return np.array(
+        healpy.pix2vec(aph.npix_to_nside(len(h)), px, nest=nest), dtype=np.float32
```

Comment:
the equivalent to `pix2vec` is `healpix_to_xyz`

Reply:
It doesn't exist yet, apparently (I can see it in the docs, but not in my local version of astropy-healpix)

Reply:
weird! I looked at the source for the `pix2vec` to find it, but maybe the source is ahead of the last release.